docs/installation.rst

```diff
@@ -68,6 +68,8 @@ initialize development environment: ::
     cd incubator-superset
     # you can run this command everytime you need to start superset now:
     docker-compose up
+    # when the 'up' command is ready, run the following command once to inititalise the default admin:admin user
```

Comment:
This should actually be run in the "init" container:

```
  superset-init:
    build: *superset-build
    command: ["/app/docker-init.sh"]
    env_file: docker/.env
    depends_on: *superset-depends-on
    volumes: *superset-volumes
```

Reply:
Maybe add some verbiage around this only needing to be done in the event that you want to re-init your setup?